modules/imgcodecs/src/exif.hpp

```diff
@@ -154,7 +154,7 @@ enum ImageOrientation
  * Usage example for getting the orientation of the image:
  *
  *      @code
- *      ExifReader reader(fileName);
+ *      ExifReader reader(std::ifstream);
```

Comment:
This code can not compile.

Reply:
Can you please mention the error, the code builds on my laptop

Reply:
Code of this snippet or OpenCV code with this comment?

Right answer should look like:
```
ExifReader reader(std::ifstream(fileName, std::ifstream::in | std::ifstream::binary));
```

Reply:
Yes this is the code that i used to test this error:
Code
#include<iostream>
#include<fstream>
#include<exif.cpp>
#include<exif.hpp>
#include<precomp.hpp>

using namespace cv;

int main(){
    std::ifstream f("/home/shiwam/Documents/opencv_test_images/shiva.jpg",std::ios_base::in | std::ios_base::binary);
    ExifReader reader(f);
    if(reader.parse()){
        int orientation = reader.getTag(ORIENTATION).field_u16;
        std::cout<<orientation;
    }else{
        std::cout<<"Failed";
    }

    

    return 0;
}

output:
1

Should I update the pr?